.github/workflows/pr-title.yml

```
@@ -43,8 +43,8 @@ jobs:
         with:
           fetch-depth: 0
       - name: changed files for plugin
-        id: plugin
-        uses: tj-actions/changed-files@v36
+        id: plugin/
```

actionlint (1.7.4)

invalid step ID "plugin/". step ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)